sql/pipelette.sql

```diff
@@ -0,0 +1,33 @@
+CREATE DATABASE pipelette;
+use pipelette;
+CREATE TABLE `user` (
+    `id` INT NOT NULL AUTO_INCREMENT,
+    `firstname` VARCHAR(100) NOT NULL,
+    `lastname` VARCHAR(100) NOT NULL,
+    `register` DATE NOT NULL,
```

Comment:
convention is to call it `created_at`, and you may prefer DATETIME instead of just DATE ;)